be/src/pipeline/exec/exchange_sink_operator.cpp

```diff
@@ -456,21 +449,16 @@ void ExchangeSinkLocalState::register_channels(
 }
 
 Status ExchangeSinkLocalState::get_next_available_buffer(
-        vectorized::BroadcastPBlockHolder** holder) {
+        std::shared_ptr<vectorized::BroadcastPBlockHolder>* holder) {
```

Comment:
```suggestion
        std::shared_ptr<vectorized::BroadcastPBlockHolder>& holder) {
```

Reply:
We'd better keep the pattern. If the parameter is the output parameter, then use pointer not reference.